test/chat_api/accounts_test.exs

```diff
@@ -1,33 +1,35 @@
 defmodule ChatApi.AccountsTest do
+  import ChatApi.Factory
   use ChatApi.DataCase, async: true
 
   alias ChatApi.{Accounts, WidgetSettings}
 
   describe "accounts" do
     alias ChatApi.Accounts.Account
 
-    @valid_attrs %{company_name: "some company_name"}
-    @update_attrs %{company_name: "some updated company_name"}
-    @invalid_attrs %{company_name: nil}
+    @valid_attrs params_for(:account)
+    @update_attrs params_for(:account, company_name: "updated company name")
+    @invalid_attrs params_for(:invalid_account)
```

Comment:
would this work if we did `params_for(:account, company_name: nil)`?

I'm not sure if we actually need to define fixtures for every "invalid" type 🤔

Reply:
invalid_attrs params_for(:account, "")  would work ofcourse nil value would skip the field